service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1052,12 +1052,12 @@ private boolean dontSendLinkToFondyIf(int sumToPay, Certificate certificate) {
 
     private void checkAmountOfBagsIfCourierLimitByAmountOfBag(TariffsInfo courierLocation, Integer countOfBigBag) {
         if (CourierLimit.LIMIT_BY_AMOUNT_OF_BAG.equals(courierLocation.getCourierLimit())
-            && courierLocation.getMinAmountOfBigBags() > countOfBigBag) {
+            && courierLocation.getMinQuantity() > countOfBigBag) {
             throw new BadRequestException(
-                NOT_ENOUGH_BIG_BAGS_EXCEPTION + courierLocation.getMinAmountOfBigBags());
+                NOT_ENOUGH_BIG_BAGS_EXCEPTION + courierLocation.getMaxQuantity());
```

Comment:
Previously we used `minAmountOfBigBags` in the message and now you are using `maxQuantity`

Reply:
done